lib/sidekiq/processor.rb

```diff
@@ -119,18 +119,21 @@ def handle_fetch_exception(ex)
       nil
     end
 
+    def log_context(job_hash)
```

Comment:
While we're doing this refactoring, it seems like this method is a logging concern that should go into Sidekiq::Logging.